steps/perfecto-upload/0.0.1/step.yml

```diff
@@ -0,0 +1,74 @@
+title: perfecto-upload
```

Comment:
Please use a more UX friendly name here, for example `Perfecto Upload`. This is the name the users will see in the workflow editor step selector.

Reply:
Thanks. Have done